### PR TITLE
User schema

### PR DESCRIPTION
**Pull request by MuhammedMagdyy**

### CODE REVIEW

File: src/database/prisma/schema.prisma [hunk 1]
```diff
@@ -16,3 +16,25 @@ model Hello {
...
 }
+
+model User {
+  id             Int      @id @default(autoincrement())
+  email          String   @unique
+  password       String
+  firstName      String
+  lastName       String
+  phone          String?  @unique
+  birthdate      DateTime
```
Comment (kerolloz) [inline src/database/prisma/schema.prisma:27]: Are you going to require email & password & first name & last name & birthdate in the first step of the registration?

Comment (MuhammedMagdyy) [reply src/database/prisma/schema.prisma:27]: Yes, I think it's the required information for registration.
I've implemented the model as we did in ERD.

File: src/database/prisma/schema.prisma [hunk 2]
```diff
@@ -16,3 +16,25 @@ model Hello {
...
+  likes          String?
+  favoriteClubs  String?
+  preferredFoods String?
+  hobbies        String?
+  Role           Role     @default(USER)
```
Comment (kerolloz) [inline src/database/prisma/schema.prisma:32]: Why `Role` ?

Comment (MuhammedMagdyy) [reply src/database/prisma/schema.prisma:32]: We talked about it on Tuesday, We said that the user can (admin, user) can create events, picks, or something like this.
If it's not required (for now) I can fix it.

**Comment by mohamadelhinamy** [comment]
looks good but please do not forget to write PR descriptions for the reviewers

**Review by mohamadelhinamy — APPROVED** [review]

**Review by kerolloz — APPROVED** [review]

**Pull request merged**